packages/core/src/routes/interaction/verifications/mfa-verification.ts

```diff
@@ -0,0 +1,86 @@
+import { InteractionEvent, MfaFactor, MfaPolicy } from '@logto/schemas';
+
+import RequestError from '#src/errors/RequestError/index.js';
+import type TenantContext from '#src/tenants/TenantContext.js';
+import assertThat from '#src/utils/assert-that.js';
+
+import { type WithInteractionDetailsContext } from '../middleware/koa-interaction-details.js';
+import { type WithInteractionSieContext } from '../middleware/koa-interaction-sie.js';
+import {
+  type VerifiedSignInInteractionResult,
+  type VerifiedInteractionResult,
+  type VerifiedRegisterInteractionResult,
+} from '../types/index.js';
+
+export const verifyBindMfa = async (
```

Comment:
this method is used to verify when binding a NEW mfa method? maybe add a comment, the name is not that straightforward